amazon-eks-nodegroup.yaml

```diff
@@ -178,23 +176,19 @@ Resources:
   NodeInstanceProfile:
     Type: AWS::IAM::InstanceProfile
     Properties:
-      Path: "/"
       Roles:
-      - !Ref NodeInstanceRole
+        - !Ref NodeInstanceRole
 
   NodeInstanceRole:
     Type: AWS::IAM::Role
     Properties:
       AssumeRolePolicyDocument:
-        Version: '2012-10-17'
+        Version: 2012-10-17
         Statement:
-        - Effect: Allow
-          Principal:
-            Service:
-            - ec2.amazonaws.com
-          Action:
-          - sts:AssumeRole
-      Path: "/"
```

Comment:
Please add the `Path` back. While this is the default, changing this value will replace the role and could cause a disruptive change

Reply:
Path: "/" re-included